src/omd.mli

```diff
@@ -46,6 +46,48 @@ type 'attr block =
 type doc = attributes block list
 (** A markdown document *)
 
+val txt : ?attrs:attributes -> string -> attributes inline
+
+val em : ?attrs:attributes -> attributes inline -> attributes inline
+
+val strong : ?attrs:attributes -> attributes inline -> attributes inline
+
+val code : ?attrs:attributes -> string -> attributes inline
+
+val hard_break : attributes -> attributes inline
+
+val soft_break : attributes -> attributes inline
```

Comment:
Would that be better to have a single `br` constructor with a `soft` argument? Something like:

```ocaml
val br : ?soft:bool -> attributes -> attributes inline
```

Reply:
I think it makes sense to have to separate constructs for this, matching the AST. It doesn't seem to cost anything in terms of maintenance or readability, and it means the user has to make less mappings in their head :)

Reply:
Valid points, thanks for the suggestion 🤗